lib/builder.dart

```diff
@@ -1,63 +1,79 @@
 import 'dart:async';
+import 'dart:io';
+
 import 'package:build/build.dart';
 import 'package:fast_i18n/src/generator.dart';
 import 'package:fast_i18n/src/model.dart';
 import 'package:fast_i18n/src/parser_json.dart';
 import 'package:fast_i18n/utils.dart';
 import 'package:glob/glob.dart';
 
-Builder i18nBuilder(BuilderOptions options) => I18nBuilder();
+Builder i18nBuilder(BuilderOptions options) => I18nBuilder(options);
 
 class I18nBuilder implements Builder {
+  I18nBuilder(this.options);
+
+  static const String defaultFilePattern = '.i18n.json';
+
+  final BuilderOptions options;
+
   bool _generated = false;
 
+  String get filesPattern => options.config['files_pattern'] ?? defaultFilePattern;
+
   @override
   FutureOr<void> build(BuildStep buildStep) async {
-    if (buildStep.inputId.pathSegments.last == 'config.i18n.json' ||
-        buildStep.inputId.pathSegments.last.indexOf('_') != -1) return;
+    String directoryInPath = options.config['directory_in'];
+    String baseLocale = options.config['base_locale'];
+    String maps = options.config['maps'];
```

Comment:
`List<String> maps` instead of `String maps`?

Reply:
Fixed